diffrax/__init__.py

```diff
@@ -40,6 +41,7 @@
 from .solution import is_event, is_okay, is_successful, RESULTS, Solution
 from .solver import (
     AbstractAdaptiveSolver,
+    AbstractANSR,
```

Comment:
I'd suggest switching from `AbstractANSR` to `AbstractAdditiveSRK`, which I think is probably a clearer name.